### ISSUE TITLE
Feature: Start on boot, and start menu button

### ISSUE THREAD

**Issue by BradKML** [ltGuillaume/Redshift-Tray#18]
Version: https://community.chocolatey.org/packages/rstray
Currently, the Redshift Tray program does not start on boot, and it is inconvenient to search for the program and find the right items in the start menu.

**Comment by ltguillaume** [comment]
I am not managing or testing the Chocolatey package, but does this mean that the autorun option from the settings doesn't work when using Chocolatey?

**Comment by BradKML** [comment]
That is correct, and that sometimes it will not launch when you click on it by searching the EXE file. This is why it is better to have it as a start menu option.

**Comment by ltguillaume** [comment]
@tunisiano187 @pauby Do you have any idea what's causing these 2 issues?

**Comment by tunisiano187** [comment]
I'm using the silent args, maybe there is an option for that?
Or did you have a command to execute after installation? I can add it to the install.

**Comment by ltguillaume** [comment]
I don't use Chocolatey. I just wanted to get you together so that hopefully you guys can reproduce and tell me what is actually going on if you think it's up to me to fix it.

I have no issue whatsoever running RSTray, or with autorun, so I don't know what to do.

**Comment by ltguillaume** [comment]
@BrandonKMLee @tunisiano187 
If it's what I think it is, then [v2.2.2](https://github.com/ltGuillaume/Redshift-Tray/releases/tag/2.2.2) will fix this.
